tests/templates/test_integration.py

```diff
@@ -145,7 +145,7 @@
                               (qml.templates.UCCSD,
                                {'weights': [3.90575761, -1.89772083, -1.36689032]},
                                {'wires': [0, 1, 2, 3], 's_wires': [[0, 1, 2], [1, 2, 3]],
-                                'd_wires': [[[0, 1], [2, 3]]], 'init_state':np.array([1, 1, 0, 0])},
+                                'd_wires': [[[0, 1], [2, 3]]], 'init_state':np.array([1, 1, 0, 0], requires_grad=False)},
```

Comment:
I see, was that one of the problems of that notorious error which repeated itself in the test?

Reply:
Yep, `qml.BasisState` was complaining since it was trying to differentiate itself